sandbox/apparmor/profile.go

```diff
@@ -51,7 +53,40 @@ const (
 	SkipKernelLoad AaParserFlags = 1 << iota
 )
 
-var runtimeNumCPU = runtime.NumCPU
+var (
+	runtimeNumCPU = runtime.NumCPU
+
+	osutilIsHomeUsingNFS        = osutil.IsHomeUsingNFS
+	osutilIsRootWritableOverlay = osutil.IsRootWritableOverlay
+)
+
```

Comment:
most of the new code in this file doesn't seem to have matching unit tests in the package? do we need some tests copied over from the backend?

Reply:
Nice catch, I'll look into this